sdk/resources/mgmt/src/main/java/com/azure/management/resources/fluentcore/arm/AzureConfigurable.java

```diff
@@ -96,4 +99,28 @@
      * @return the configurable object itself for chaining
      */
     T withProxy(Proxy proxy);
+
+    /**
+     * Sets the credential scope.
+     *
+     * @param scope the credential scope
+     * @return the configurable object itself for chaining
+     */
+    T withScope(String scope);
```

Comment:
what's this?

Reply:
The credential scope can be used in AuthenticationPolicy. @ChenTanyi should be able to suggest whether we plan to keep this.

Reply:
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#security-scheme-object

Currently we are using hardcoded `/.default`